jdaviz/core/data_formats.py

```diff
@@ -272,3 +273,35 @@ def identify_helper(filename, ext=1):
         return 'imviz'
 
     raise ValueError(f"No helper could be auto-identified for {filename}.")
+
+
+def open(filename, show=True, **kwargs):
+    '''
+    Automatically detect the correct configuration based on a given file,
+    load the data, and display the configuration
+
+    Parameters
+    ----------
+    filename : str (path-like)
+        Name for a local data file.
+    show : bool
+        Determines whether to immediately show the application
```

Comment:
Ok I'm going to be super careful here because I got it wrong above 😅

The helper is a convenience class that exposes certain methods to the user. The helper CONTAINS an `Application` which is actually being rendered using `helper.show()`. So I believe `Application` is appropriate here

Reply:
Thanks for the explanation! I genuinely wasn't sure what the correct terminology was, so I went with the:
> ...maybe? 
😄